### ISSUE TITLE
Swagger definition invalid

### ISSUE THREAD

**Issue by damianoneill** [OpenNetworkingFoundation/TAPI#317]
The definition published here https://github.com/OpenNetworkingFoundation/TAPI/blob/develop/SWAGGER/tapi-connectivity%402018-02-16.swagger

is invalid json, this can be confirmed as follows: 

```
$ curl -sL https://raw.githubusercontent.com/OpenNetworkingFoundation/TAPI/develop/SWAGGER/tapi-connectivity%402018-02-16.swagger | jq
```

The issue is that ine 16986 is a duplicate of 16985, removing 16985 makes it valid.

**Comment by damianoneill** [comment]
@bzeuner  Is there any update on this?

**Comment by bzeuner** [comment]
Hallo @damianoneill,
I cannot really help here and would like to point you to @karthik-sethuraman

**Comment by karthik-sethuraman** [comment]
Hi,
You are cirrect and it is fixed now by #320. Currently there is no plan to release further 2.0.x versions, but
for the next TAPI release 2.1, we are planning to use updated RESTConf mapping, which should not have this issue.
